Agoric-CLI.md

```diff
@@ -0,0 +1,90 @@
+## Agoric CLI
+
+You use the Agoric CLI to install dependencies, initialize, deploy, and start Agoric projects. Add one of five command names to `agoric` to specify what command to run. **Note**: Required argument names need not be given, only their values. Optional argument names must be given, along with their values. See the command's examples if you're not sure if a name is needed.
+
+In general, you want to issue Agoric CLI commands in this order:
+1. `agoric init`
+2. `agoric install`
+3. `agoric start` (Usually with `--reset`)
+4. `agoric deploy`
+
+Use `agoric help` whenever you need help with one of the above Agoric CLI commands.
+
+## `agoric init`
+- **Function**: 
+  - Creates a new Dapp directory named `<project>` with contents copied from the `dapp-template` argument template.
+- **Required Arguments**:
+    - `project`: Name of the new project to initialize.
+- **Optional Arguments**:
+    - `--dapp-template <name>`: Use the template specified by `<name>`. Defaults to the demo Dapp `dapp-simple-exchange`.
+    - `--dapp-base <base-url>`: Look under this directory for the Dapp template. Defaults to `git://github.com/Agoric/`
+- **Examples**:
+  - Creates a directory named `demo` with contents copied from the default `dapp-template` value `dapp-simple-exchange`.
+	- `agoric init demo`
+  - Creates a directory named `my-first-contract` with contents copied from the default `dapp-template` value `dapp-simple-exchange`.
+	  - `agoric init my-first-contract`
+  - Creates a directory named `my-contract` with contents copied from a template named `dapp-skeleton` located under the URL `file:///home/contracts`
+	  - `agoric init my-contract --dapp-template dapp-skeleton --dapp-base file:///home/contracts` 
+
+## `agoric install`
+- **Function**:
+  - Install Dapp JavaScript dependencies. This may take a while.
+- **Required Arguments**:
+  - None.
+- **Optional Arguments**:
+  - None.
+- **Examples**:
+  - Installs Dapp JavaScript dependencies
+	- `agoric install`
+
+## `agoric start`
+- **Function**:
+  - Run an Agoric VM on which contracts will run.
+- **Required Arguments**:
+  - `[profile]`
```

Comment:
This defaults to `dev` for development mode.  You can also set it to `testnet` to connect to the current testnet.